packages/flutter_tools/templates/app/pubspec.yaml.tmpl

```diff
@@ -41,6 +41,15 @@ dependencies:
     # the parent directory to use the current plugin's version.
     path: ../
 {{/withPluginHook}}
+{{#withFfiPluginHook}}
+  {{pluginProjectName}}:
+    # When depending on this package from a real application you should use:
+    #   {{pluginProjectName}}: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
+    path: ../
+{{/withFfiPluginHook}}
```

Comment:
Can we rename `withPluginHook` to `withPlatformChannelPluginHook`, and keep `withPluginHook` as the "or" of these two, so we don't need to have two identical copies of this?